tests/bootstrap.php

```diff
@@ -1,54 +1,37 @@
 <?php
+/**
+ * This file handles initializing MediaWiki stack and setting global flags
+ * before handing over execution to PHPUnit that executes the tests
+ * provided in its configuration file set via the command line.
+ */
+
+error_reporting( E_ALL );
+ini_set( 'display_errors', 1 );
+
 $wgRunningUnitTests = true; // don't include DevBoxSettings when running unit tests
 $wgDevelEnvironment = true;
+$wgAnnotateTestSpeed = ( getenv( 'ANNOTATE_TEST_SPEED' ) === '1' );
 
-class FakeCache {
-	public function __call($a, $b) {
-		return '';
-	}
-}
+require_once __DIR__ . '/../maintenance/Maintenance.php';
 
-require_once __DIR__ . '/../maintenance/commandLine.inc';
+class PhpUnit extends Maintenance {
+	public function __construct() {
+		parent::__construct();
 
-class MessageStub extends Message {
-	public function toString() {
-		throw new WikiaException('No messaging in unit tests');
+		$this->addOption( 'slow-list', 'Whether to find and mark slow tests' );
 	}
-}
 
-function wfMsgExtStub() {
-	return '';
-}
-function wfMsgRealStub() {
-	return '';
-}
-function wfGetDBStub() {
-	throw new WikiaException('No DB in unit tests');
-}
+	public function execute() {
+		// TODO: Broken. Fix.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)